exporters/otlp/internal/transform/metric_test.go

```diff
@@ -278,7 +279,14 @@ func TestSumInt64DataPoints(t *testing.T) {
 	assert.NoError(t, s.Update(context.Background(), metric.Number(1), &desc))
 	require.NoError(t, s.SynchronizedMove(ckpt, &desc))
 	record := export.NewRecord(&desc, &labels, nil, ckpt.Aggregation(), intervalStart, intervalEnd)
-	if m, err := sum(record, ckpt.(aggregation.Sum)); assert.NoError(t, err) {
+	sum, ok := ckpt.(aggregation.Sum)
+	if !ok {
+		t.Errorf("ckpt is not an aggregation.Sum: %T", ckpt)
+	}
```

Comment:
Should this be a fatal error here so the following isn't executed?

```suggestion
	require.True(t, ok, "ckpt is not an aggregation.Sum: %T", ckpt)
```

Reply:
Done